Scientific_workflows/C3_Validation/fc_percentile_check.ipynb

```diff
@@ -0,0 +1,630 @@
+{
```

Comment:
<p>is it ok to have this in here?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GeoscienceAustralia/dea-notebooks/pull/853/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>It's a public bucket, I'd assume it's ok.</p>

Reply:
<p><code>dev</code>  is merely how I named my credential, credential itself is a secret not disclosed. It's of no use to anybody else.</p>